examples/any-type/any_type.bal

```diff
@@ -11,7 +11,7 @@ function main(string... args) {
     any a = 5;
     io:println(a);
 
-    // First, the variable of the `any` type needs to be casted to the required type (`int` in this example) as shown here. More information about type casting is available in the next section.
+    // First, the variable of the `any` type needs to be cast to the required type (`int` in this example) as shown here. More information about type casting is available in the next section.
```

Comment:
@samgnaniah Pls review this.

Reply:
Looks good, but what do we mean by 'next section' here? When BBE is rendered to ballerina.io, we don't have sections. Shall we remove that line?

Reply:
Here, it seems to refer to the next example, which, in the site, happens to be the type conversion example. Agreed, referring to a "next section" sounds weird in the context of BBE. Will explicitly mentioning the type conversion example be good enough or should we get rid of that sentence altogether?